script.js

```diff
@@ -0,0 +1,177 @@
+//   image: 'images/program_icon_03.png',
+//   title: 'COLLAB LEARNING',
+//   discretion: 'Learn through pair programming, group projects,and project reviews.',
+
+// }, {
+//   image: 'images/program_icon_01.png',
+//   title: 'GLOBAL CHANCE',
+//   discretion: 'Connect with global jobs after gaining experience working on a remote team.',
+// }, {
+//   image: 'images/program_icon_05.png',
+//   title: 'SUPPORT AND LIABLE',
+//
+// },
+// ];
+// const story = [{
+//   name:"kevin Mwangi",
+//   image:"images/Screenshot (17).png"
+//   ,link:"https://www.microverse.org/blog/a-career-with-microsoft-how-microverse-helped-pave-the-way"
+// },
+// {name:"Gabriela Cruz Olivera",
+// image:"images/Screenshot (11).png",
+// link:"https://www.microverse.org/blog/join-the-global-job-market-as-a-web-developer"}
+// ];
+
+const team = document.querySelector('.team-box');
+// const spons = document.querySelector('.backed-by');
+// const box = document.querySelector('.main-big-box');
+// const success = document.querySelector(".success-img");
+
+function boredmembers() {
+  teammembers.forEach((teammembers) => {
+    team.innerHTML += `
+      <div class="mobile-version">
+      <div class="team-profile">
+                <div class="image-box">
+                    <img class="square-img" src="/images/Screenshot (12).png" alt="">
+                <img class="team-image"src="${teammembers.image}" alt="${teammembers.name} image">
+          </div>
+          <div class="team-name"><h4>${teammembers.name}</h4>
+            <p>${teammembers.profession}</p></div>
+            </div>
+            <p>${teammembers.discretion}</p></div>
+
+            <div class="desktop-version">
+      <div class="team-profile">
+                <div class="image-box">
+                    <img class="square-img" src="images/Screenshot (12).png" alt="">
+                <img class="team-image"src="${teammembers.image}" alt="${teammembers.name} image">
+          </div>
+          <div class="minebox">
+          <div class="team-name"><h4>${teammembers.name}</h4>
+            <p>${teammembers.profession}</p>
+            </div>
+            <p>${teammembers.discretion}</p></div></div></div>
+    `;
+  });
+}
+boredmembers();
+
+// function backup() {
+//   sponsor.forEach((sponsor) => {
+//     spons.innerHTML += `
+
+//       <img class="sponser-img"src="${sponsor.logo}" alt="${sponsor.name}">
+//     `;
+//   });
+// }
+// backup();
+
+// function experience() {
+//   learn.forEach((learn) => {
+//     box.innerHTML += `
+
+//     <div class="main-box">
+//             <img src="${learn.image}" alt="${learn.title}">
+//             <h4>${learn.title}</h4>
+//             <p>${learn.discretion}</p>
+//         </div>
+//   `;
+//   });
+// }
+// experience();
+
+// function winstory() {
+//   story.forEach((story) => {
+//     success.innerHTML += `
+
+//     <a href="${story.link}"><img src="${story.image}" alt="${story.name}"></a>
+//   `;
+// });
+// }
+// winstory()
```

Comment:
- [ ] I kindly suggest that your remove commented out code from your codebase to make your work neater.